### PR TITLE
Ansible Installation

### PR DESCRIPTION
**Pull request by nitinrajput1997**
Signed-off-by: Nitin Rajput<nitinrajput706021@gmail.com>

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ kubectl exec -it ${NMS_POD} -- yarn setAdminPassword magma-test admin admin
 
 Setup Ansible:
 ```bash
-sudo apt install -y ansible
+sudo apt update
+sudo apt install software-properties-common
+sudo add-apt-repository --yes --update ppa:ansible/ansible
+sudo apt install ansible
 ansible-galaxy collection install shubhamtatvamasi.magma --force-with-deps
```
Comment (ShubhamTatvamasi) [inline deploy/README.md:46]: I think that this command won't work after we have installed latest ansible. Can you confirm?

Comment (nitinrajput1997) [reply deploy/README.md:46]: yes

Comment (nitinrajput1997) [reply deploy/README.md:46]: this is only for installation if there is no ansible installed previously. Suppose, if ansible is installed but we want to upgrade it to latest version then we have to add one more command above this to remove previously installed ansible and above command then install latest version

Comment (ShubhamTatvamasi) [reply deploy/README.md:46]: I am talking about `collection install` command.

Comment (nitinrajput1997) [reply deploy/README.md:46]: ok. Yes it won't work

Comment (ShubhamTatvamasi) [reply deploy/README.md:46]: Then update it too.

Comment (nitinrajput1997) [reply deploy/README.md:46]: ok

Comment (nitinrajput1997) [reply deploy/README.md:46]: hi,  _collection install_ work properly.

**Review by ShubhamTatvamasi — APPROVED** [review]

**Pull request merged**